modules/core/src/main/scala/gem/enum/GmosRoi.scala

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) 2016-2017 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package gem
+package enum
+
+import scalaz.syntax.equal._
+import scalaz.std.string._
+
+/**
+ * Enumerated type for GMOS ROI (region of interest).
+ * @group Enumerations (Generated)
+ */
+sealed abstract class GmosRoi(
```

Comment:
This is just renamed from `GmosBuiltinRoi` since it now includes a `Custom` option.  I removed the specific ROI descriptions because we don't use them in OCS2.